packages/streamid/src/commit-id.ts

```diff
@@ -17,11 +18,11 @@ import { StreamRef } from './stream-ref.js'
  * @see CommitID#bytes
  */
 function fromBytes(bytes: Uint8Array): CommitID {
-  const result = fromBytesNoThrow(bytes)
-  if (result instanceof Error) {
-    throw result
+  const parsed = parsing.fromBytes(bytes, 'CommitID')
+  if (parsed.kind === 'commit-id') {
+    return new CommitID(parsed.type, parsed.genesis, parsed.commit)
   }
-  return result
+  throw new Error(`No commit information provided`)
```

Comment:
```suggestion
  throw new Error(`Error while parsing CommitID from byte array: no commit information provided`)
```

Reply:
The error messages from these errors can show in logs without much context about what was happening when the error was thrown.  We won't always have a stack trace.  So it's important to always try to put as much context into error messages as possible